src/app/static/src/app/store/projects/actions.ts

```diff
@@ -227,12 +227,12 @@ export const actions: ActionTree<ProjectsState, RootState> & ProjectsActions = {
 
     async renameProject(context, projectPayload: projectPayload) {
         const {state, dispatch} = context;
-        const {projectId, name} = projectPayload
+        const {projectId, name, note} = projectPayload
 
         await api<ProjectsMutations, ErrorsMutation>(context)
             .ignoreSuccess()
             .withError(`errors/${ErrorsMutation.ErrorAdded}` as ErrorsMutation, true)
-            .postAndReturn(`/project/${projectId}/rename`, qs.stringify({name}))
+            .postAndReturn(`/project/${projectId}/rename`, qs.stringify({name: name, note: note}))
```

Comment:
I know this is tested indirectly but you should also add a test to [actions.test.ts](https://github.com/mrc-ide/hint/blob/master/src/app/static/src/tests/projects/actions.test.ts#L646) to ensure that a mock POST is made, given a payload containing a note.